src/schemas/21-526EZ-allclaims/schema.js

```diff
@@ -594,78 +594,93 @@ const schema = {
     },
     form0781: {
       type: 'object',
-      incident: {
-        type: 'array',
-        items: {
-          type: 'object',
-          properties: {
-            personalAssault: {
-              type: 'boolean'
-            },
-            medalsCitations: {
-              type: 'string'
-            },
-            incidentDate: {
-              $ref: '#/definitions/date'
-            },
-            incidentLocation: {
-              type: 'string'
-            },
-            incidentDescription: {
-              type: 'string'
-            },
-            unitAssigned: {
-              type: 'string'
-            },
-            unitAssignedDates: {
-              $ref: '#/definitions/dateRange'
-            },
-            remarks: {
-              type: 'string'
-            },
-            additionalChanges: {
-              type: 'string'
-            },
-            personInvolved: {
-              type: 'array',
-              items: {
-                type: 'object',
-                name: {
-                  $ref: '#/definitions/fullName'
-                },
-                rank: {
-                  type: 'string'
-                },
-                injuryDeath: {
-                  type: 'string',
-                  enum: [
-                    'Killed in Action',
-                    'Killed Non-Battle',
-                    'Wounded in Action',
-                    'Injured Non-Battle',
-                    'Other'
-                  ]
-                },
-                injuryDeathOther: {
-                  type: 'string'
-                },
-                injuryDeathDate: {
-                  $ref: '#/definitions/date'
-                },
-                unitAssigned: {
-                  type: 'string'
+      properties: {
+        "remarks": {
+          "type": "string"
```

Comment:
remove double quotes from the key and use single quote for the value

Reply:
Thanks!